ms-campus-kid-grades/.gitignore

```diff
@@ -1 +1,2 @@
-**/__pycache__/
\ No newline at end of file
+**/__pycache__/
+/virtual
```

Comment:
Please add a new line at the end of the file.

Reply:
why?

Reply:
It is required for git that all files should have a new line at the end.